x/ecocredit/client/tx.go

```diff
@@ -172,6 +173,34 @@ Parameters:
 	}
 }
 
+func txCancel() *cobra.Command {
+	return &cobra.Command{
+		Use:   "cancel [credits]",
+		Short: "Cancels a specified amounts of credits from the account of the transaction author (--from)",
+		Long: `Cancels a specified amounts of credits from the account of the transaction author (--from)
+
+Parameters:
+  credits:  YAML encoded credit list. Note: numerical values must be written in strings.
```

Comment:
Let's not do YAML in the CLI. Can we think of some other way to do this?

Reply:
Yeah, I'll have a think about something else. The YAML was awkward during manual testing.

Reply:
It's slightly harder than with coins as batch denoms look like `18AV53K/1b1vrHF`, possibly having digits in front. We could do something like `10:18AV53K/1b1vrHF,100:18AV53K/1b1vrHF`?

Reply:
There's also YAML in the `create-batch` command. Would you prefer to remove that as well?